### DISCUSSION TITLE
How do I secure the RESTful Model Endpoint ?

### DISCUSSION THREAD

**Discussion by jiyer2016** [bentoml/BentoML#1250] [Q&A] [answered]
What options does one have to secure the RESTful Model Endpoint running on a container ?

**Comment by yubozhao** [comment, marked as answer]
I think the common ways to secure REST API are: basic auth, JWT(json web token), and oauth.
Right now, you can try to put an authentication server that redirects valid requests to the API server. 

What's your current setup?

**Reply by jiyer2016** [reply]
I going with the jwt approach. Thanks.
